templates/zerver/help/view-zulip-version.md

```diff
@@ -0,0 +1,46 @@
+# View Zulip version
+
+## Zulip server and web app version
+
+If your organization self-hosts Zulip, the server administrator will need to
+[upgrade Zulip][upgrade-zulip] in order for you to experience the latest
+improvements. You can check which [version][changelog] of the Zulip server and
+web app your organization is using.
```

Comment:
We should perhaps add a sentence somewhere here that it's often useful to include what Zulip version you're using in support tickets?  I guess that should clarify that just mentioning you're using Zulip Cloud (or your organization URL) is sufficient for this information for cloud users.

Reply:
I spent some time debating this, and decided against it to avoid adding friction to the process of reporting a problem.